pkgs/development/python-modules/pprintpp/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, nose, parameterized }:
```

```suggestion
{ lib, buildPythonPackage, fetchPypi, python, nose, parameterized }:
```

for `${python.interpreter}`